README.md

```diff
@@ -1,11 +1,50 @@
 # codacy-clang-tidy
 
+**Warning: This is a work in progress, it is not yet usable.**
+
 A standalone tool that converts [Clang-Tidy](https://clang.llvm.org/extra/clang-tidy/)
 reports to Codacy's format. It allows the integration of Clang-Tidy into your Codacy workflow.
```

Comment:
I think we could expand a bit better on why codacy-clang-tidy is needed an how it is used.

From the command-line examples below I could understand that the output from the clang-tidy tool must be piped to the codacy-clang-tidy for the format conversion to take place. But if this could be explained sooner, perhaps even with a very simple diagram, it would help users understand how it all works.

Reply:
I've created a task to create that documentation, since it is generic for all "custom-tools". That documentation will be release before this tool is ready for use, I will update the readme with a link to it